internal/oci/runtime_oci.go

```diff
@@ -145,6 +145,11 @@ func (r *runtimeOCI) CreateContainer(ctx context.Context, c *Container, cgroupPa
 	if r.config.LogSizeMax >= 0 {
 		args = append(args, "--log-size-max", strconv.FormatInt(r.config.LogSizeMax, 10))
 	}
+	if r.handler.NoSyncLog {
+		log.Warnf(ctx, "NoSyncLog is enabled for %s (%s), this can lead to lost log data",
```

Comment:
this will log for each container creation. can we instead log it when we validate it in `pkg/config/config.go`?

Reply:
Sure, this has the advantage of being very explicit of each use of it (even without using `ps` to verify).
I'll move it to the validation.